lib/scrollyteller/scrollyteller.ts

```diff
@@ -69,25 +70,38 @@ export interface ScrollyOptions {
 
 export class Scrollyteller {
   private parachart: ParaChart;
+  private paraview: ParaView;
   private steps!: NodeListOf<Element>;
 
-  constructor(chartID?: string) {
-    const chart = chartID ?
-      document.getElementById(chartID) as ParaChart | null
-      : document.querySelector('para-chart-ai') as ParaChart | null;
-      
-    if (!chart) {
-      throw new Error(
-        `Scrollyteller requires a ParaChart element. ${
-          chartID 
-            ? `No element found with ID "${chartID}"` 
-            : 'No para-chart-ai element found on page'
-        }`
-      );
+
+  constructor(
+    parachart: ParaChart,
+    paraview: ParaView,
+    chartID?: string,
+  ) {
+    // HACK: needed to assign something to this.parachart
+    this.parachart = parachart;
+    this.paraview = paraview;
+    if (this.paraview.store.settings.scrollytelling.isScrollytellingEnabled) {
```

Comment:
```suggestion
    if (this.parachart.paraview.store.settings.scrollytelling.isScrollytellingEnabled) {
```
Could we just use the `paraView` getter here?